src/main/java/ca/uhn/fhir/jpa/starter/dotbase/PlainSystemProviderR4.java

```diff
@@ -79,4 +85,11 @@ public IBundleProvider latestSection(HttpServletRequest theRequest, RequestDetai
     return sectionOperation.latestSection(sectionTypeEnum, patientId);
   }
 
+  @Operation(name = "$last-edited-section", idempotent = true)
```

Comment:
The name of this operation is misleading. I would suggest leaving out the 'section' part. This is because `last-edited-section` tells something about the implementation of this operation rather then its resulting information.
```suggestion
  @Operation(name = "$last-edited", idempotent = true)
```